tinygrad/codegen/assembly_rdna.py

```diff
@@ -116,11 +115,8 @@ def reg_out(x):
         else:
           ins.append(f"{'s_' if out.scalar else 'v_'}mov_b32 {reg_out(out)}, {arg}")
       elif uop == UOps.ALU:
-        if arg == BinaryOps.CMPLT:
-          if out.scalar:
-            ins.append(f"s_{alu[arg]}_{dtype_to_rdnatype[out.dtype]} {', '.join(reg_in(x) if x.__class__ is Register else str(x) for x in vin)}")
-          else:
-            ins.append(f"v_cmp_lt_{dtype_to_rdnatype[out.dtype]} vcc, {', '.join(reg_in(x) if x.__class__ is Register else str(x) for x in vin)}")
```

Comment:
From what I could tell from the docs, `v_cmp_*` instructions always write to `vcc` and don't take it as an argument.

Reply:
I think it might depend on instruction encoding; might have something to do with presence/lack of `_e32` or `_e64` suffix like `v_cmp_eq_f32_e32` or something.

I'm not an expert on this assembly language, so if someone more knowledgeable could check I'd appreciate it.  I've not encountered any compilation errors after this change with anything I've tested, though.